streaming/src/main/scala/org/apache/spark/streaming/util/FileBasedWriteAheadLogSegment.scala

```diff
@@ -17,4 +17,5 @@
 package org.apache.spark.streaming.util
 
 /** Class for representing a segment of data in a write ahead log file */
-private[streaming] case class WriteAheadLogFileSegment (path: String, offset: Long, length: Int)
+private[streaming] case class FileBasedWriteAheadLogSegment(path: String, offset: Long, length: Int)
```

Comment:
I have been thinking the same. The consistent way would be `FileBasedWriteAheadLogRecordHandle`. That is super super long. :( So I kept it as segment. I think its better of the two options.